raiden/tests/integration/network/transport/rif_comms/process.py

```diff
@@ -0,0 +1,50 @@
+import os
+import signal
+import subprocess
+import time
+from pathlib import Path
+from subprocess import Popen
+
+import psutil
+
+# FIXME: this should be a dependency within the project
+# path to the RIF Comms Node repo
+RIF_COMMS_PATH = Path(__file__).parents[7].joinpath('rif-communications-pubsub-node')
```

Comment:
This is relative path? i didn't understood this but we should read that from an environment variable and if doesn't exists assume a relative or absolute path right?

Reply:
> This is relative path? i didn't understood this but we should read that from an environment variable and if doesn't exists assume a relative or absolute path right?

yes. this means "from the current file, move up a dir 7 times". this path matches the containing folder of the `lumino` repo, which _should_ be the same folder that contains the `rif-comms` repo (see _Requirements_ section).

i hope this clears it up.

Reply:
So we never want to change lumino comms to another path? i'm just asking because maybe for deployment this could change and be customized that's all

Reply:
> So we never want to change lumino comms to another path? i'm just asking because maybe for deployment this could change and be customized that's all

of course. as it says in the `FIXME`, comms should be a test env dependency installed through `pip` (although i'm not 100% sure on how this would work) and then located through environment variables.